airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -219,76 +173,61 @@ def collect(self):
             "airflow_num_queued_tasks", "Airflow Number of Queued Tasks"
         )
 
-        num_queued_tasks = get_num_queued_tasks(TaskInstance)
+        num_queued_tasks = get_num_queued_tasks()
         num_queued_tasks_metric.add_metric([], num_queued_tasks)
         yield num_queued_tasks_metric
 
-        sla_miss_metric = GaugeMetricFamily(
-            "airflow_sla_miss",
```

Comment:
Just to confirm, the sla miss metric will not be supported under Astonomer?

Reply:
Here is the part one which is to keep the out of box metrics from exporter, and create two endpoints to provide the information needed for DDNS.

Will have second PR that gathers data from these two new endpoints and combine the alert delay rules to produce both sla_miss and unmonitored metrics from there. 

This is to avoid storing data in Airflow DB (required by Astronomer).

Reply:
Got it - makes sense!